inc/meta-box-html.php

```diff
@@ -11,78 +11,92 @@ class HTML {
 		'hidden' => array( 'nonce', 'hidden', 'separator', 'fieldset' ),
 		);
 
-	public function draw( $field, $form_id = NULL ) {
+	public function draw( $field, $set_id = NULL ) {
 		if ( empty( $field ) ) {
-			return new WP_Error( 'field_required', 'You need to pass a field array to this method. You passed a '. gettype( $field ) . ' .');
+			wp_die( 'field_required', 'You need to pass a field array to this method. You passed a '. gettype( $field ) . ' .');
 		}
-		?><div class="cms-toolkit-wrapper<?php if (isset( $field['class'] )) { echo ' ' . esc_attr( $field['class'] ); } ?>"><?php
-		if ( $field['type'] !== 'formset' and isset( $field['title'] ) ) {
-			?><h4 id="<?php echo "{$field['meta_key']}"; ?>" ><?php
+		?><div class="cms-toolkit-wrapper<?php echo isset( $field['class'] ) ?  ' ' . esc_attr( $field['class'] ) : null; ?>"><?php
```

Comment:
I guess we don't have an official position against the ternary operator here, but why bother? The `else` condition has no effect, so isn't it more efficient to just have an `if` statement?